railties/lib/rails/generators/rails/app/templates/.rubocop.yml.tt

```diff
@@ -0,0 +1,2 @@
+# Omakase Ruby styling for Rails
+inherit_gem: { rubocop-rails-omakase: rubocop.yml }
```

Comment:
Missing EOF

Reply:
Its also missing the frozen-string-literal, because this file is generated it falls under the rules of `rubocop-rails-omakase` and not the framework's [settings](https://github.com/rails/rails/blob/main/.rubocop.yml).